docs/source/guide/configuration.rst

```diff
@@ -48,9 +50,83 @@ To set these configuration options, create a ``Config`` object with the options
 
     client = boto3.client('kinesis', config=my_config)
 
+
+Using proxies
+~~~~~~~~~~~~~
+Boto3 supports using proxies as intermediaries between your code and AWS. Proxies may provide any number of functions, from filtering to security and firewalls to privacy assurance.
+
+
+.. _specify_proxies:
+
+Specifying proxy servers
+''''''''''''''''''''''''
+
+You can specify proxy servers to be used when communicating between specific endpoints, or when
+using specific protocols. The ``proxies`` option in the ``Config`` object is a dictionary that maps
+the protocol name or endpoint URL to the address of a proxy server.
+
+.. code-block:: python
+
+    import boto3
+    from botocore.config import Config
+
+    proxy_list = {
+        'http': 'proxy.example.com:6502',
```

Comment:
We should also have a scheme for the proxies as well since it's not guaranteed to be the same as the key.

Reply:
We also should probably avoid putting examples with externally controlled domains, especially for things like proxies.

Reply:
OK, I copied the code from the tests in the patch, and since they didn't use schemes, I left them out. But I've fixed it in my current draft. And I've changed the domains to be amazon ones. Thanks for the catch! :)